### PR TITLE
Merge Reg data mapping

### PR DESCRIPTION
**Pull request by congnc-anz**
Add reg data mapping

### CODE REVIEW

File: fum_flows/models/stage/Reg_Data_Mapping.sql [hunk 1]
```diff
@@ -0,0 +1,170 @@
+WITH
+reg_mapping as (
+  select 
+    FUM as fum,
+    BU as bu,
+    DIVISION_NAME as division_name,
+    Retail_Mapping,
```
Comment (khoinguyen-anz) [inline fum_flows/models/stage/Reg_Data_Mapping.sql:7]: Change column name
Retail_Mapping as retail_mapping

File: fum_flows/models/stage/Reg_Data_Mapping.sql [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    CASE
+      WHEN DEAL_SUBTYPE LIKE '%V2%' THEN 'V2'
+      ELSE DEAL_SUBTYPE
+    END AS sub_product_code
+  from {{ ref("reg_data_mapping")}}
+),
+
+reg_mapping_TPD as (
+  select 
+    *,
```
Comment (khoinguyen-anz) [inline fum_flows/models/stage/Reg_Data_Mapping.sql:23]: Please write clear out the column name from each CTE until union joins

**Comment by congnc-anz** [comment]
Changes base on comment has been applied @khoinguyen-anz

**Comment by khoinguyen-anz** [comment]
Improved by another pull request

**Pull request closed without merging**